lib/Controllers/create.js

```diff
@@ -30,6 +57,9 @@ Create.prototype.write = function(req, res, context) {
         res.header('Location', location);
       }
 
+      // Reload for that all field are present
+      return instance.reload();
```

Comment:
secondly, forcing reloads for a simple crud operation vs one with includes should be considered. A forced reload with associations is likely to cause even more overhead on the wire (potentially many sql statements you don't necessarily care about).